### PR TITLE
Improvements to slides and activity section names

### PR DESCRIPTION
**Pull request by bethanyaconnor**
- Look for markdown headings on their own and make them the name of the next activity section. While still imperfect, this produces more activity sections with names
- Major improvements to the way the slide icon is handled. The slide and tip icons are strange because in curriculum builder, they functioned as a placeholder for an icon, whereas in levelbuilder they're more of a property of a section of text. Meaning, there will never be a one-to-one mapping here.

CSP lessons saw the most improvement because they make good use of the slide icon which was apparently quite broken.
Screenshot 2020-11-19 at 8 33 31 AM

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by davidsbailey — APPROVED** [review]
I can't say I understand the nitty gritty here as well as you and Dani, but LGTM if you need to get this merged before Dani has a chance to review. Once merged, if you can push this PR to the adhoc, I can take care of re-running the import. here are the steps I plan to execute:

1. destroy all lesson groups in CSP 2020 scripts
2. re-seed CSP 2020 scripts
3. re-import CSP 2020 scripts

**Comment by bethanyaconnor** [comment]
Yep that's perfect! Thanks Dave!

And yeah, the diff is a bit hard to read for this because I changed slides _and_ added titles so they look like replacements of each other, which they're not. I essentially moved towards treating slides as tiplinks (which is how the CB markdown renderer treats them) and then checked for them more often (within remarks, for example).

**Comment by bethanyaconnor** [comment]
The drone failure is in UI tests, which are likely due to the sauce labs outage from this morning. Nothing in this PR is covered by UI tests so I'm not going to bother rerunning

**Pull request merged**
